pkg/mounter/ossfs_utils.go

```diff
@@ -0,0 +1,127 @@
+package mounter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/alibabacloud-go/tea/tea"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+)
+
+// keep consitent with RAM response
+var secretRefKeysToParse []string = []string{
+	"AccessKeyId",
+	"AccessKeySecret",
+	"Expiration",
+	"SecurityToken",
+}
+
+func getPasswdSecretVolume(secretRef, nodeName, volumeId string) (secret *corev1.SecretVolumeSource) {
+	passwdFilename := "passwd-ossfs"
+	if secretRef == "" {
+		secret = &corev1.SecretVolumeSource{
+			SecretName: OssfsCredentialSecretName,
+			Items: []corev1.KeyToPath{
+				{
+					Key:  fmt.Sprintf("%s.%s", nodeName, volumeId),
+					Path: passwdFilename,
+					Mode: tea.Int32(0600),
+				},
+			},
+			Optional: tea.Bool(true),
+		}
+	} else {
+		items := []corev1.KeyToPath{}
+		for _, key := range secretRefKeysToParse {
+			item := corev1.KeyToPath{
+				Key:  key,
+				Path: fmt.Sprintf("%s/%s", passwdFilename, key),
+				Mode: tea.Int32(0600),
```

Comment:
Use defaultMode for this? Also. I don't expect importing `tea` here.

Do we really need to append `passwd-ossfs/` to the Path? If we drop the prefix, we don't need all these `KeyToPath`. And we can set Optional to false, to ensure the secret do exist as expected before ossfs start.

Reply:
Ossfs restricts passwd-file mode without any others permission, 600 is a normal mode as official document.
I removed the secret brought from secretRef, and for csi-ossfs-creds secret maintained by CSI @tydra-wang ptal